crates/bevy_render/src/view/visibility/render_layers.rs

```diff
@@ -115,6 +115,45 @@ impl RenderLayers {
     pub fn bits(&self) -> u32 {
         self.0
     }
+
+    /// Returns the set of [layers](Layer) shared by two instances of [`RenderLayers`].
+    pub const fn and(self, other: Self) -> Self {
+        let mask = self.0 & other.0;
+        Self(mask)
+    }
+
+    /// Returns all [layers](Layer) included in either instance of [`RenderLayers`].
+    pub const fn or(self, other: Self) -> Self {
```

Comment:
I considered calling these `intersection` and `union` for clarity, but I wasn't sure what to call xor using that naming convention

Reply:
This would be the symmetric difference or disjunctive union. Pretty obscure so not sure it's worth switching. Note that Python does have a symmetric difference operation on sets. https://en.m.wikipedia.org/wiki/Symmetric_difference

Reply:
It probably is worth switching, at least to avoid potential confusion with the `and` method. I imagine some people may think `a.and(b)` means "all the layers from a and b"